raijin_scripts/deploy/dea-env/environment.yaml

```diff
@@ -1,91 +1,195 @@
 name: dea-env
 channels:
 - conda-forge
-- defaults
-- nodefaults
 dependencies:
+- affine >= 2.2.1
```

Comment:
What's the plan for keeping these dependency versions up to date?

Reply:
Conditional versions for the `dependency` was removed except few dependencies such as celery, rasterio, rasterstats, distributed, dask, and python versions.

Reply:
Thanks Damien for the comments. I have made changes to the code as per the comments and pushed the changes back to repo.